mido/midifiles/midifiles.py

```diff
@@ -399,18 +399,18 @@ def play(self, meta_messages=False):
 
         You will receive copies of the original messages, so you can
         safely modify them without ruining the tracks.
-
         """
-        sleep = time.sleep
-        now = time.time
-        t0 = now()
-        file_time = 0
+        t0 = time.time()
+        input_time = 0.0
 
         for msg in self:
-            file_time += msg.time
-            duration_to_next_event = file_time - (now() - t0)
-            if (duration_to_next_event > 0):
-                sleep(duration_to_next_event)
+            input_time += msg.time
+
+            playback_time = time.time() - t0
```

Comment:
`t0` looks a bit out of place. Would `start_time` be a better name? (Maybe there's some convention here I'm unaware of that speaks against it.)

Reply:
No problem for me to rename `t0` to `start_time`. Upcoming ...

Reply:
Cool! Thanks.